esignet-mock.properties

```diff
@@ -401,56 +401,36 @@ mosip.esignet.vci.authn.allowed-audiences={ '${mosip.esignet.domain.url}${server
 
 mosip.esignet.cnonce-expire-seconds=40
 mosip.esignet.vci.supported.jwt-proof-alg={'RS256'}
-mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.domain.url}',         \
+mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.vci.identifier}', 	\
   'credential_endpoint': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', \
   'credentials_supported': {{\
   'format': 'ldp_vc',\
   'id': 'InsuranceCredential', \
-  'scope' : 'mock_identity_vc_ldp',\
+  'scope' : 'sunbird_rc_insurance_vc_ldp',\
```

Comment:
can we have both the mock and sunbird here as 2 items in the credentials_supported list ?

Reply:
as of now it is not possible i think

Reply:
added the mock sample vc